db/migrate/20201117080045_create_likes.rb

```diff
@@ -6,5 +6,7 @@ def change
 
       t.timestamps
     end
+
+    add_index :likes, %i[user_id likeable_type likeable_id], unique: true
```

Comment:
you can't modify migration if it's already merged

Reply:
Should I create new migration?

Reply:
yes, please